docs/3.0.0-beta.x/deployment/nginx-proxy.md

```diff
@@ -65,6 +83,218 @@ server {
+
+Alternatively for the admin, you can replace the proxy instead with serving the admin `build` folder directly from Nginx, such centralizing the admin but load balancing the backend APIs. The example for this is not shown, but it would likely be something you would build into your CI/CD platform.
+
+::: warning
+Please note that this config is not focused on the frontend hosting, you will most likely need to adjust this to your frontend software requirements, it is only being shown here as an example.
+:::
+
+---
+
+Example API Domain: `example.com/api`
+
+Example Admin Domain: `example.com/dashboard`
+
+Path: `/etc/nginx/sites-available/strapi.conf`
+
+```
+server {
+    # Listen HTTP
+    listen 80;
+    server_name example.com;
+
+    # Redirect HTTP to HTTPS
+    return 301 https://$host$request_uri;
+}
+
+server {
+    # Listen HTTPS
+    listen 443 ssl;
+    server_name example.com;
+
+    # SSL config
+    ssl_certificate /path/to/your/certificate/file;
+    ssl_certificate_key /path/to/your/certificate/key;
+
+    # Static Root
+    location / {
+        root /var/www/html;
+    }
+
+    # Strapi API
+    location /api/ {
+        rewrite ^/api/(.*)$ /$1 break;
+        proxy_pass http://strapi;
+        proxy_http_version 1.1;
+        proxy_set_header X-Forwarded-Host $host;
+        proxy_set_header X-Forwarded-Server $host;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+        proxy_set_header Host $http_host;
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_set_header Connection "Upgrade";
+        proxy_pass_request_headers on;
+    }
+
+    # Strapi Dashboard
+    location /dashboard {
+        proxy_pass http://strapi/dashboard;
+        proxy_http_version 1.1;
+        proxy_set_header X-Forwarded-Host $host;
+        proxy_set_header X-Forwarded-Server $host;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+        proxy_set_header Host $http_host;
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_set_header Connection "Upgrade";
+        proxy_pass_request_headers on;
+    }
+}
+```
+
+::::
+
+:::::
+
+### Strapi Server
+
+In order to take full advantage of a proxied Strapi application you will need to configure Strapi to make it aware of the upstream proxy. Like with the above Nginx configurations there are 3 matching examples. To read more about this server configuration file please see the [server configuration concept](../concepts/configurations.md#server) documentation.
+
+::::: tabs
+
+:::: tab Sub-Domain
+
+#### Sub-Domain Strapi config
+
+---
+
+Example Domain: `api.example.com`
+
+Path: `config/server.js`
```

Comment:
```diff
- Path: `config/server.js`
+ **Path —** `config/server.js`
```

Reply:
Absolutely I'll fix all of them in a bit